flink-table-store-spark/src/main/java/org/apache/flink/table/store/spark/SparkSource.java

```diff
@@ -69,7 +70,8 @@ public Table getTable(
                 Configuration.fromMap(SparkCaseSensitiveConverter.convert(options));
         FileSystem.initialize(
                 configuration, PluginUtils.createPluginManagerFromRootFolder(configuration));
-        return new SparkTable(FileStoreTableFactory.create(Configuration.fromMap(options)));
+        return new SparkTable(
+                FileStoreTableFactory.create(Configuration.fromMap(options)), Lock.emptyFactory());
```

Comment:
If there is no lock or no need for lock, does this need to create empty lock?

Reply:
I think it is just for simplify. Avoid null check.